tests/cypress/latest/support/commands.ts

```diff
@@ -114,11 +114,11 @@ Cypress.Commands.add('createMachReg', (
         // In rare case, we might want to test upgrading from staging to dev
         utils.isUpgradeOsChannel('dev') ? cy.contains('ISO x86_64 (unstable)').click(): null;
       } else {
-          cy.contains('ISO x86_64 v1.2.2')
+          cy.contains('ISO x86_64 v1.2.3')
```

Comment:
I don't know if there's a way to use a variable here, to avoid changing the code each time the version is updated (maybe easier to change a variable in the GH workflow or in a config file).

But if it's doable keep this for another PR.